docs/context/ref/Action.md

```diff
@@ -0,0 +1,65 @@
+---
+id: Action
+sidebar_label: Action
+title: Action
+hide_title: true
+---
+# `Action`
+
+A representation of an FDC3 Action (specified via a Context or Context & Intent) that can be inserted inside another object, 
+for example a chat message.
+
+The action may be completed by calling `fdc3.raiseIntent()` with the specified Intent and Context, or, if only a context is specified, by calling `fdc3.raiseIntentForContext()` (which the Desktop Agent will resolve by presenting the user with a list of available Intents for the Context).
+
+## Type
+
+`fdc3.action`
+
+## Schema
+
+https://fdc3.finos.org/schemas/next/action.schema.json
+
+## Details
+
+| Property          | Type                                      | Required | Example Value           |
+|-------------------|-------------------------------------------|----------|-------------------------|
+| `type`            | string                                    | Yes      | `'fdc3.action'`  |
+| `title`           | string                                    | Yes      | `'Click to view Chart'` |
+| `context`         | string                                    | Yes      | See Below               |
+| `intent`          | string                                    | No       | `'ViewChart'`           |
```

Comment:
It occurs to me that an `app` target property might also be useful here as you might want to specify a specific app to perform the action. See arguments to `raiseIntent`.

Reply:
I can confirm that we've ended up needing the `app` argument in the work on notifications - it should have type `AppIdentifier` so that it can accept `appId` and optionally `instanceId` so that it can identify a particular app (and less important for this usecase, but needed in notifications) a specific instance of it.